examples/aldp.py

```diff
@@ -342,13 +359,50 @@ def initial_sampler():
         buffer = PrioritisedReplayBuffer(dim=ndim, max_length=rb_config['max_length'] * batch_size,
                                          min_sample_length=rb_config['min_length'] * batch_size,
                                          initial_sampler=initial_sampler, device=str(device))
-        buffer_sample = None
         # Set target distribution
         def ais_target_log_prob(x):
             return 2 * model.target_distribution.log_prob(x) - model.flow.log_prob(x)
         model.annealed_importance_sampler.target_log_prob = ais_target_log_prob
 else:
     use_rb = False
+    if filter_chirality_train:
```

Comment:
I had to add these to be able to filter for chirality during training also for the baseline models.